packages/@aws-cdk/aws-batch-alpha/lib/ecs-container-definition.ts

```diff
@@ -896,6 +896,13 @@ export interface IEcsFargateContainerDefinition extends IEcsContainerDefinition
    * @default LATEST
    */
   readonly fargatePlatformVersion?: ecs.FargatePlatformVersion;
+
+  /**
+   * The size for ephemeral storage.
+   *
+   * @default - no storage
```

Comment:
Are we sure this is the default?

Reply:
Kaizen makes a good point, we should do this: https://docs.aws.amazon.com/batch/latest/APIReference/API_EphemeralStorage.html

https://aws.amazon.com/about-aws/whats-new/2023/03/aws-batch-configurable-ephemeral-storage-fargate/

the minimum is 21 GB, and the max is 200 GB. The default is 20 GB

Reply:
Sounds good, thanks for catching @kaizencc! @comcalvi should the default be 21 GB instead of 20? With a minimum of 21 GB, wouldn't a default of 20 GB always be flagged by the validation?